jupyter_server/services/contents/handlers.py

```diff
@@ -136,26 +137,27 @@ async def get(self, path=""):
             raise web.HTTPError(400, "Content %r is invalid" % content_str)
         content = int(content_str or "")
 
-        md5_str = self.get_query_argument("md5", default="0")
-        if md5_str not in {"0", "1"}:
-            raise web.HTTPError(400, "Content %r is invalid" % md5_str)
-        md5 = int(md5_str or "")
+        hash_str = self.get_query_argument("hash", default="0")
+        if hash_str not in {"0", "1"}:
+            raise web.HTTPError(400, "Content %r is invalid" % hash_str)
+        require_hash = int(hash_str or "")
 
         if not cm.allow_hidden and await ensure_async(cm.is_hidden(path)):
             await self._finish_error(
                 HTTPStatus.NOT_FOUND, f"file or directory {path!r} does not exist"
             )
+        kwargs = {
+            "path": path,
+            "type": type,
+            "format": format,
+            "content": content,
+        }
+        if cm.support_hash:
+            kwargs["require_hash"] = require_hash
```

Comment:
If the ContentsManager does not declare itself to support `hash`(by setting `support_hash` to True), handler will never pass `require_hash` argument to the `ContentsManager.get`.

I don't want to use `hash` for the argument as it's a built-in function.